apps/base-docs/docs/account-abstraction-with-base-paymaster/intro-to-account-abstraction.md

```diff
@@ -0,0 +1,163 @@
+
+By the way, you'll often see the terms "smart contract wallet", "smart contract account" or just "smart account" used interchangeably.
+
+At first glance, you may be asking yourself if using smart contract accounts solve our problem. What's the difference between a smart contract account (or contract account) and an "Externally Owner Account"?
+
+According to [https://ethereum.org/en/developers/docs/accounts/#key-differences](https://ethereum.org/en/developers/docs/accounts/#key-differences), the differences are:
+
+**Externally Owned Account**
+
+- Creating an account costs nothing
+- Can initiate transactions
+- Transactions between externally-owned accounts can only be ETH/token transfers
+- Made up of a cryptographic pair of keys: public and private keys that control account activities
+
+**Contract Accounts**
+
+- Creating a contract has a cost because you're using network storage
+- Can only send transactions in response to receiving a transaction
+- Transactions from an external account to a contract account can trigger code which can execute many different actions, such as transferring tokens or even creating a new contract
+- Contract accounts don't have private keys. Instead, they are controlled by the logic of the smart contract code
+
+As we stated before, smart contract wallets _would_ solve our problems, **but**, according to the information above, they can't initiate transactions. And since they cannot initiate transactions, we still need EOAs and those EOAs still need to pay for gas with ETH.
+
+There are a couple options to solve for this:
+
+1. Change the protocol - hard fork! This would be a massive undertaking and most would agree that it is not a viable option.
+
+2. Change transactions, upstream (ERC-4337).
+
+In a nutshell, ERC-4337 doesn't change how Ethereum transactions work. Cryptographically signed instructions from accounts still initiate transactions to update the state of the Ethereum network.
+
+What ERC-4337 changes is everything _upstream_ of that signed transaction. It does this by introducting a new "user intent layer" that acts as a proxy for an EOA. This allows users to initiate transactions, with highly customizable smart contract wallets, but without using an EOA. Ethereum still recieves what it recieved before - signed transactions.
+
+The result is a better experience for the user without changing the Ethereum protocol.
+
+---
+
+## How Account Abstraction Works
+
+With typical Ethereum transactions an EOA initiates and signs a transaction. That transaction is sent to Ethereum's Public Mempool, is validated and added to a block, onchain.
+
+Note that the following steps are primarily happening _before_ any of that.
+
+#### 1. **Smart Contract Wallet Creation**
+
+First, a new smart contract wallet must be created for a user. This wallet is owned by it's creator who is designated as the "signer". This signer must validate any of it's future operations.
```

Comment:
How is it owned by the creator?  And by creator do we mean the user, or is it owned by the entity that creates it for the user?